packages/react-components/src/components/LoadingSpinner.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { Spinner } from '@fluentui/react';
+import React from 'react';
+import { loadingSpinnerContainerStyle } from './styles/LoadingSpinner.style';
+
+/**
+ * @public
+ */
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface LoadingSpinnerProps {}
+
+const spinnerStyles = {
+  circle: {
+    height: '100px',
+    width: '100px',
+    borderWidth: '5px'
+  },
+  label: {
```

Comment:
label here has an explicit value, but I don't see a label in the spinner, can this be removed?

Reply:
Mmmm nope not needed. I can remove it, unless we wanna show a label like "Loading...", wdyt?